src/docpage.js

```diff
@@ -10,13 +10,131 @@ export default class DocPage extends React.Component {
     })
   }
 
+
   render() {
     let doc = (this.state || {}).doc
     if(! doc) return <p>loading...</p>
 
+    let files = doc.data.files && doc.data.files.map((item, index) => {
+      return (
+        <tr key={index}>
+          <td>
+            {item.id ?
+              <a href={`./${item.id}`}>{item.filename}</a> :
+              <span>{item.filename}</span>
+            }
+          </td>
+          <td className="text-muted">{item.content_type}</td>
+          <td>
+            {item.id ?
+              <a href={`./${item.id}/raw/`} title="Original file">
+                <i className="fa fa-file-o"></i>
+              </a> :
+              <code>-- broken link --</code>}
+          </td>
+        </tr>
+      )
+    })
+
+
     return (
-      <pre><code>{JSON.stringify(doc, null, 2)}</code></pre>
+      <div className="card doc-page">
+        <div className="card-block">
```

Comment:
Insert buttons around here for the original document, the OCR document (if it exists) and the parent document (Up). You can find the snoop implementation [here for the Up link](https://github.com/hoover/snoop/blob/master/snoop/templates/document.html#L57), [here for the Original File link](https://github.com/hoover/snoop/blob/master/snoop/templates/document.html#L61) and so on.